src/frontend/data/stores/useVideo/index.tsx

```diff
@@ -0,0 +1,21 @@
+import create from 'zustand';
+
+import { Video } from '../../../types/tracks';
+import { Nullable } from '../../../utils/types';
+import { appData } from '../../appData';
+
+interface State {
+  getVideo: (video: Nullable<Video>) => Video;
+  videos: {
+    [id: string]: Video;
+  };
+}
+
+export const [useVideo] = create<State>((set, get) => ({
+  getVideo: (video: Nullable<Video>) => {
```

Comment:
Ditto here. Consumers can just as well index on the `videos` object?

Reply:
instead of having this helper, we could simply do this in the component directly ? This is what you mean ?